requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyOpenSSL
-pulpcore>=3.3.0
+pulpcore>=3.4
```

Comment:
We still want 3.3.0 to be the oldest one.

Reply:
oh! I didn't notice it, I will fix it